android/src/main/java/io/rownd/android/models/network/AppConfig.kt

```diff
@@ -100,15 +103,33 @@ data class AppConfigConfig(
 
 @Serializable
 data class HubConfig(
-    var auth: HubAuthConfig = HubAuthConfig()
+    var auth: HubAuthConfig = HubAuthConfig(),
+    var customizations: HubCustomizationsConfig? = HubCustomizationsConfig(),
+    @SerialName("custom_styles")
+    @JsonNames("customStyles")
```

Comment:
where are we using `custom_styles` vs `customStyles`?

Reply:
I removed the JsonNames from the AppConfig for customStyles and fontFamily